tests/chainer_tests/links_tests/rnn_tests/test_link_n_step_rnn.py

```diff
@@ -413,4 +414,74 @@ def test_n_cells(self):
         assert self.rnn.n_cells == 1
 
 
+@testing.parameterize(
+    *testing.product(
+        {
+            'dtype': [numpy.float32, numpy.float64],
+            'initializer': ['random', 'zero'],
+            'activation_type': ['tanh', 'relu'],
+            'use_bi_direction': [True, False]
+        }
+    )
+)
+class TestInitialization(unittest.TestCase):
+    def setUp(self):
+        if self.initializer == 'random':
+            initializer = initializers.GlorotUniform()
+
+        elif self.initializer == 'zero':
+            initializer = initializers.constant.Zero()
+
+        self.initialW = numpy.zeros((10, 10), dtype=self.dtype)
+        self.initial_bias = numpy.zeros((10, 1), dtype=self.dtype)
+
+        initializer(self.initialW)
+        initializer(self.initial_bias)
```

Comment:
How about `self.initialW = numpy.random.uniform(-1, 1, (10, 10)).astype(self.dtype)`?

To test initializers, `setUp` should have `self.link = link(..., initialW=initializer, initial_bias=initializer)`.

Reply:
1. Updated to initialize `self.initialW` and `self.initial_bias` by `numpy.random.uniform`.

2. The current implementation is based on [this approach](https://github.com/chainer/chainer/pull/5807#discussion_r242031702) since I couldn't find a way to reproduce the same initialization results in `setUp` and `NStepRNN.__init__` (setting seed before/after `initializer(self.initialW)` didn't work). Could you please let me know if you have any ideas?